openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java

```diff
@@ -348,7 +349,7 @@ else if (ALT_ROUTE.equalsIgnoreCase(algoStr))
 
 			return altPaths;
 
-		} catch (IllegalArgumentException ex) {
+		} catch (Exception ex) {
```

Comment:
Just as a clarification - do we want to be doing a catch-all Exception rather than more specific one?

Reply:
no, I can't even tell you why that changed anymore... reverted.

Reply:
and regarding the single conditional line braces, I'm all for deciding for one style and doing it everywhere.